client/src/main/scala/client/MonixSampleClient.scala

```diff
@@ -17,7 +18,8 @@ object MonifuSampleClient extends js.JSApp {
     val line4 = new DataConsumer(200.millis, -9826395057397L, doBackPressure = false)
       .collect { case s: Signal => s }
 
-    Observable.combineLatest(line1, line2, line3, line4)
+    Observable
+      .combineLatest4(line1, line2, line3, line4)
       .subscribe(new Graph("lineChart"))
```

Comment:
NOTE: this should NOT be `unsafeSubscribeFn`, because here is the call-site (this is the user).